kernel/kernel-api/src/main/java/io/delta/kernel/internal/TableConfig.java

```diff
@@ -272,6 +272,33 @@ public class TableConfig<T> {
           "needs to be a boolean.",
           true);
 
+  /**
+   * The value that enables uniform exports to Iceberg for {@linkplain
+   * #UNIVERSAL_FORMAT_ENABLED_FORMATS}.
+   *
+   * <p>{@link #ICEBERG_COMPAT_V2_ENABLED but also be set to true} to fully enable this feature.
+   */
+  public static final String UNIVERSAL_FORMAT_ICEBERG = "iceberg";
```

Comment:
Can this be in a nested class of some sorts?

Something like `TableConfig.UniversalFormat.FORMAT_ICEBERG`?

I wonder if that scope would make it more readable, especially as this is a public value and `TableConfig. UNIVERSAL_FORMAT_ICEBERG` is just a string -- not actually a `TableConfig`.

What do you think?

Reply:
I went with UniversalFormats.  Name spacing seems reasonable.  We could also make an enum or remove the FORMAT_ prefix, don't have a strong preference and happy to do whatever is idiomatic for Kernel